3rdparty/utfcpp/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+add_library(utfcpp INTERFACE)
```

Comment:
Why don't we include this as a submodule?

Reply:
It's not the recommended way by upstream. Also, they are discontinuing the CMake project.

Reply:
In that case I would suggest forking the library under our organization and then using that as a submodule.

Reply:
Done.